mne_icalabel/megnet/label_components.py

```diff
@@ -0,0 +1,120 @@
+import os.path as op
+
+import numpy as np
+import onnxruntime as ort
+from mne.io import BaseRaw
+from mne.preprocessing import ICA
+from numpy.typing import NDArray
+
+from .features import get_megnet_features
+
+
+def megnet_label_components(
+    raw: BaseRaw,
+    ica: ICA,
+    model_path: str = op.join("assets", "network", "megnet.onnx"),
```

Comment:
Do we need to expose the model path? I presume the model is always going to use this model?

Perhaps the API can look similar to ICLabel, and add the `inplace` argument?

https://github.com/mne-tools/mne-icalabel/blob/b2fc448dd2a5718eeeefee14d051525f2cb4b209/mne_icalabel/iclabel/label_components.py#L12